gstreamer-rtsp-server/src/r_t_s_p_token.rs

```diff
@@ -1,12 +1,33 @@
-use RTSPToken;
-use glib::value::ToSendValue;
-use gst;
 use ffi;
+use gst_ffi;
+use gst;
+use glib;
+use glib::StaticType;
+use glib::value::ToSendValue;
 use glib::translate::*;
 
-impl RTSPToken {
-    pub fn new(values: &[(&str, &ToSendValue)]) -> RTSPToken {
-        let mut token = RTSPToken::new_empty();
+use gst::miniobject::{MiniObject, GstRc};
+
+pub trait GstRcExt<T: MiniObject> {
+    fn new_empty() -> Self;
```

Comment:
Hrm this is quite ugly! So there's a problem implementing miniobjects outside the main crate.

With your solution, can you call `RTSPToken::new_empty()`? The name should probably be `GstRcRTSPTokenExt` though, we would need the same thing for other external miniobjects and traits with the same name, while not conflicting, cause weird issues sometimes.

Reply:
Done